doc/release-notes.md

```diff
@@ -10,7 +10,7 @@ and performance improvements, as well as updated translations.
 
 Please report bugs using the issue tracker at GitHub:
 
-  <https://github.com/RavenProject/Ravencoin/issues>
```

Comment:
This seems like it should still be RavenProject/Ravencoin

Reply:
Yep, my mistake while changing the others which referenced specific past issues.